client/blocks/comment-detail/style.scss

```diff
@@ -17,6 +17,14 @@
 		z-index: 1;
 	}
 
+	// If the comment is pending and collapsed, color it yellow.
+	&:not( .is-approved ):not( .is-expanded ) {
```

Comment:
This is caused by my poor CSS classes design, but I'm actually not really into this whole `:not` part.

I'd rather add a `.is-pending` and `.is-closed`(-contract? -shrunk?) [here](https://github.com/Automattic/wp-calypso/blob/master/client/blocks/comment-detail/index.jsx#L132-L140), and update the stylesheet accordingly.